test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt

```diff
@@ -1252,6 +1246,32 @@ AndroidArgs
         val parsedYml = AndroidArgs.load(yaml)
         runBlocking { runAndroidTests(parsedYml) }
     }
+
+    /**
+     * Double directory check for regressive issue protect, first to check directory
+     * really not exist on disk (when issue come back and directory exists, assert not throws),
+     * second for checking directory not created after load args
+     * it's should be enough
+     */
+    @Test
+    fun `should not throw if directory not exists`() {
```

Comment:
Comments should usually describe why we're doing something, not what we're doing.

> Usually comments are useful when they explain why some code exists, and should not be explaining what some code is doing. 
>
> https://github.com/google/eng-practices/blob/master/review/reviewer/looking-for.md#comments


What do you think about this?
```kotlin
   @Test
   fun `results-dir (cloud directory) should not throw if it doesn't exist locally`
```

Reply:
Thanks for this feedback! I change test name and remove my comment. :)